src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/tests/PageList.test.js

```diff
@@ -114,11 +113,10 @@ test('Should show the locales from the webspace configuration for the toolbar',
 
     const webspaceKey = observable.box('sulu');
 
-    // $FlowFixMe
-    const webspace = {
+    const webspace = ({
         key: 'sulu',
         allLocalizations: [{localization: 'en', name: 'en'}, {localization: 'de', name: 'de'}],
-    };
+    }: any);
```

Comment:
Why don't you use the `defaultWebspace` with the spread operator here, to avoid casting this to `any`? Also true for all other occurences in this file.

Reply:
because if the `webspace.localizations` property of the passed webspace is not undefined, the redirect logic of the `PageList` component will trigger 🙈

Reply:
Then I would probably set do something like this:

```javascript
const webspace = {
    ...defaultWebspace,
    localizations: undefined,
};
```

And add `// $FlowFixMe` comments where necessary.

Reply:
did it that way 🙂